apps/aecore/src/aec_tx_pool_sync.erl

```diff
@@ -332,7 +332,7 @@ do_local_action(Sync = #sync{ peer_id = PeerId }, {get, TxHashes}) ->
     Fun = fun() ->
               case aec_peer_connection:tx_pool_sync_get(PeerId, TxHashes) of
                   {ok, Txs} ->
-                      [ aec_tx_pool:push(Tx) || Tx <- Txs ],
+                      [ aec_tx_pool:push_sync(Tx) || Tx <- Txs ],
```

Comment:
Is this an attack vector? A malicious node could pollute the mempool of another node via sync?

Reply:
@hanssv i removed `push_sync/1` function in favour of `push(Tx, tx_received)`. I'm checking nonce offset only for `tx_created` event but jobs is controlling queue for it.

Reply:
We're starting to sync mempool without synced chain. Because of that we can end up with huge nonce deltas when account is not yet present in state. I created a ticket for it: [#162147646: Wait for mempool sync until chain sync is done](https://www.pivotaltracker.com/story/show/162147646)